en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,322 @@
+- [Airspeed](../config/airspeed.html)
+
+### RC-Setup
+
+[Calibrate your RC Controller](../config/radio.md) and setup the [flight mode switches](../config/flight_mode.md).
+
+### Actuator Setup
+
+:::warning
+Make sure the props are removed!
+The motors are easy to start in the actuators tab by accident.
+:::
+
+Motors, control surfaces, and other actuators are configured in the QGroundControl [Actuator Configuration & Testing](../config/actuators.md).
+
+
+#### Tilt Servos
+
+Switch the vehicle into manual mode. Either via the flight mode switch or type "commander mode manual" into the mavlink shell.
+Check if the motors are pointing upwards. If they are pointing forwards they need to be reversed.
+To do so select the checkbox under PWM-Main for each servo. Adjust the minimum or maximum value that the servo is pointing vertical up.
+Then type "commander transition" into the mavlink shell to adjust the horizontal position.
+
+![Tilt Servo adjustment](../../assets/airframes/vtol/omp_hobby_zmo_fpv/tilt-limits-01.jpg)
+
+#### Control Surfaces
+
+Check with RC-Controller, if the actuators need to be reversed.
+
+- Roll stick to the right -> Right aileron up, left aileron down
+- Pitch stick to the back (fly upwards) -> both V-tail surfaces up
+- Yaw stick to the right -> both surfaces to the right
+
+Now adjust the trim value that all the surfaces are in neutral position.
+
+![Servo trim](../../assets/airframes/vtol/omp_hobby_zmo_fpv/servo_trim.png)
+
+#### Motor Direction and Orientation
+
+Make sure the props are removed!!!
+
+- Motor 1: Front left motor should spin CW
+- Motor 2: Front right motor should spin CCW
+- Motor 3: Rear motor should spin CCW
+
+If the motor spins in the wrong directions two of the three motor wires need to be swapped. The direction can not be changed in software since the ESCs are not using dshot.
+
+## First Flight
+
+- Check CG (There are markings underneath the wing)
+- Check actuator orientations and neutral trim
+- Check tilt rotor reactions in [Stabilized mode](../flight_modes_fw/stabilized.md)
+  - Yaw the vehicle to the right (nose to the right) -> right motor should tilt down
+  - Yaw the vehicle to the left (nose to the left) -> left motor should tilt down
+- Check tilt rotor reactions in [Stabilized mode](../flight_modes_fw/stabilized.md)
+  - Roll the vehicle to the right -> Right aileron should go down
+  - Pitch the vehicle up (nose up) -> both elevons should go down
+  - Yaw the vehicle to the right (nose to the right) -> both elevons should go to the right,
+- Check kill switch
+- Arm in [Stabilized mode](../flight_modes_fw/stabilized.md) and check if motors respond to the commands, e.g. roll left increases throttle on the right motor
+- Takeoff in stabilized and make some basic maneuvers
+- If everything went without any issue, takeoff in [Position mode](../flight_modes_fw/position.md) and do a transition at around 50m. If something goes wrong switch back to MC mode as fast as possible.

```

Comment:
What about tuning? SHould we autotune?

Reply:
No we'll provide recommended PX4 params including the tuning.

Reply:
Sounds good, but what's the workflow? Are we expecting them to load airframe, then params then tidy up the actuator output mapping for how they wired it, or is this just something we provide at the end?

NOrmally we'd say "do all this, including tuning" then "here is a param file we provided earlier". Why? Because I'd like to make sure our tuning information is followable for this.

Reply:
You're right, it's a bit contradicting atm. I've proposed a workflow that should work in https://github.com/PX4/PX4-user_guide/pull/2772/files#r1346984309